server/game/core/card/UpgradeCard.ts

```diff
@@ -53,6 +53,10 @@ export class UpgradeCard extends UpgradeCardParent {
         return actions;
     }
 
+    public override getSummary(activePlayer: Player, hideWhenFaceup: boolean) {
+        return { ...super.getSummary(activePlayer, hideWhenFaceup), parentCardId: this._parentCard ? this._parentCard.uuid : null, aspects: this.aspects };
```

Comment:
Why are we handling the aspects here? Shouldn't that be a top-level attribute for every card?